upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -724,8 +724,8 @@ if [[ ${state_recorded} == "0" && $(hostname) == "ncn-m001" ]]; then
 
     # As boot parameters are added or removed, update these arrays.
     # NOTE: bootparameters_to_delete should contain keys only, nothing should have "=<value>" appended to it.
-    bootparameters_to_set=( "split_lock_detect=off" "psi=1" "rd.live.squashimg=rootfs" )
-    bootparameters_to_delete=( "rd.live.squashimg" )
+    bootparameters_to_set=( "split_lock_detect=off" "psi=1" "rd.live.squashimg=rootfs" "rd.live.overlay.thin=0" )
```

Comment:
Was this supposed to be `=1`?  Or did I mix the values up?

Reply:
It's supposed to be `=0`.